### PR TITLE
Fix N+1 queries for reviews in staging project model

### PR DESCRIPTION
**Pull request by dmarcoux**
Also use the `|` to join the arrays and remove the duplicates

### CODE REVIEW

File: src/api/app/models/staging/staging_project.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def missing_reviews
       @missing_reviews = []
       attribs = [:by_group, :by_user, :by_project, :by_package]
 
-      (requests_to_review + staged_requests).uniq.each do |request|
+      (requests_to_review | staged_requests.includes(:reviews)).each do |request|
```
Comment (Ana06) [inline src/api/app/models/staging/staging_project.rb]: :cupid:

Comment (coolo) [reply src/api/app/models/staging/staging_project.rb]: @Ana06 you're just too easy :smile:

Comment (bgeuken) [reply src/api/app/models/staging/staging_project.rb]: @dmarcoux I didn't realize that earlier... This only works when combined with fa178200e32a7cfb2897031d19bfb23ba21b176f

**Comment by bgeuken** [comment]
This is now part of #6180

**Pull request closed without merging**